### PR TITLE
learning path: add more popular links to Reference

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Added more links in the Reference section in Learning Path.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: _index.md [hunk 1]
```diff
@@ -125,6 +125,12 @@ hide_commit: true
 
 <LearningPath label="参考" icon="cloud-dev">
 
+[TiDB 配置文件](https://docs.pingcap.com/zh/tidb/dev/tidb-configuration-file)
+
+[TiDB 配置参数](https://docs.pingcap.com/zh/tidb/dev/command-line-flags-for-tidb-configuration)
```
Comment (TomShawn) [inline _index.md:130]:
```suggestion
[TiDB 命令行参数](https://docs.pingcap.com/zh/tidb/dev/command-line-flags-for-tidb-configuration)
```

File: _index.md [hunk 2]
```diff
@@ -125,6 +125,12 @@ hide_commit: true
 
 <LearningPath label="参考" icon="cloud-dev">
 
+[TiDB 配置文件](https://docs.pingcap.com/zh/tidb/dev/tidb-configuration-file)
```
Comment (TomShawn) [inline _index.md:128]:
```suggestion
[TiDB 配置文件参数](https://docs.pingcap.com/zh/tidb/dev/tidb-configuration-file)
```

File: _index.md [hunk 3]
```diff
@@ -125,6 +125,12 @@ hide_commit: true
...
+
+[TiDB 配置参数](https://docs.pingcap.com/zh/tidb/dev/command-line-flags-for-tidb-configuration)
+
+[TiDB Control](https://docs.pingcap.com/zh/tidb/dev/tidb-control)
```
Comment (TomShawn) [inline _index.md:132]:
```suggestion
[命令行工具 TiDB Control](https://docs.pingcap.com/zh/tidb/dev/tidb-control) 
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: _index.md [hunk 4]
```diff
@@ -125,6 +125,12 @@ hide_commit: true
 
 <LearningPath label="参考" icon="cloud-dev">
 
+[TiDB 配置文件参数](https://docs.pingcap.com/zh/tidb/dev/tidb-configuration-file)
+
+[TiDB 命令行参数](https://docs.pingcap.com/zh/tidb/dev/command-line-flags-for-tidb-configuration)
+
+[命令行工具 TiDB Control](https://docs.pingcap.com/zh/tidb/dev/tidb-control)
```
Comment (hfxsd) [inline _index.md:132]:
```suggestion
[TiDB Control](https://docs.pingcap.com/zh/tidb/dev/tidb-control) 
```

File: _index.md [hunk 5]
```diff
@@ -125,6 +125,12 @@ hide_commit: true
 
 <LearningPath label="参考" icon="cloud-dev">
 
+[TiDB 配置文件参数](https://docs.pingcap.com/zh/tidb/dev/tidb-configuration-file)
+
+[TiDB 命令行参数](https://docs.pingcap.com/zh/tidb/dev/command-line-flags-for-tidb-configuration)
+
+[TiDB Control](https://docs.pingcap.com/zh/tidb/dev/tidb-control)
```
Comment (hfxsd) [inline _index.md:132]:
```suggestion
[TiDB Control](https://docs.pingcap.com/zh/tidb/dev/tidb-control)  
```

**Pull request merged**